common/script_values/wc_age_values.txt

```diff
@@ -6,67 +6,67 @@
 racial_age_multiplier_value = {
 	value = 1
 	if = {
-		limit = { is_high_elf_trigger = yes }
+		limit = { is_race_trigger = { RACE = creature_high_elf } }
 		multiply = 22
 	}
 	else_if = {
-		limit = { is_night_elf_trigger = yes }
+		limit = { is_race_trigger = { RACE = creature_night_elf } }
 		multiply = 30
 	}
 	else_if = {
-		limit = { is_highborn_elf_trigger = yes }
+		limit = { is_race_trigger = { RACE = creature_highborne } }
 		multiply = 22
 	}
 	else_if = {
-		limit = { is_nightborne_elf_trigger = yes }
+		limit = { is_race_trigger = { RACE = creature_nightborne } }
 		multiply = 22
 	}
 	else_if = {
-		limit = { is_dwarf_trigger = yes }
+		limit = { is_race_trigger = { RACE = creature_dwarf } }
 		multiply = 6
 	}
 	else_if = {
-		limit = { is_gnome_trigger = yes }
+		limit = { is_race_trigger = { RACE = creature_gnome } }
 		multiply = 6
 	}
 	else_if = {
-		limit = { is_naga_trigger = yes }
+		limit = { is_race_trigger = { RACE = creature_naga } }
 		multiply = 22
 	}
 	else_if = {
-		limit = { is_satyr_trigger = yes }
+		limit = { is_race_trigger = { RACE = creature_satyr } }
 		multiply = 22
 	}
 	else_if = {
 		limit = { is_vrykul_trigger = yes }
```

Comment:
Changed all the other ones but not this one? Why?

Reply:
> Changed all the other ones but not this one? Why?

Theres kvaldir and vrykul.